lightning/src/ln/channelmanager.rs

```diff
@@ -4487,6 +4521,21 @@ where
 					peer_state.outbound_v1_channel_by_id.retain(|chan_id, chan| process_unfunded_channel_tick(chan_id, &mut chan.context, &mut chan.unfunded_context));
 					peer_state.inbound_v1_channel_by_id.retain(|chan_id, chan| process_unfunded_channel_tick(chan_id, &mut chan.context, &mut chan.unfunded_context));
 
+					for (chan_id, req) in peer_state.inbound_channel_request_by_id.iter_mut() {
+						if { req.ticks_remaining -= 1 ; req.ticks_remaining } <= 0 {
+							log_error!(self.logger, "Force-closing unaccepted inbound channel {} for not accepting in a timely manner", log_bytes!(&chan_id[..]));
+							peer_state.pending_msg_events.push(
+								events::MessageSendEvent::HandleError {
+									node_id: counterparty_node_id,
+									action: msgs::ErrorAction::SendErrorMessage {
+										msg: msgs::ErrorMessage { channel_id: chan_id.clone(), data: "Channel force-closed".to_owned() }
```

Comment:
I think we need a clearer description in the message here. I know we might not know the *real* reason for not letting the request time out, but maybe we could just say:

`"Channel force-closed due to acceptance timeout"` or something better. What do you think?

P.S. I've added the missing `MessageSendEvent` in #2496, which we I will rebase on `main` when this PR lands.

Reply:
Oh, right lol